packages/docs/src/components/BasicNiivueCanvas.jsx

```diff
@@ -1,6 +1,15 @@
-import React from "react";
+import React, { useRef, useEffect } from "react";
 import { Niivue } from "@niivue/niivue";
+import BrowserOnly from "@docusaurus/BrowserOnly";
 
+<<<<<<< HEAD
+export const NiivueApp = ({ images, nvOpts }) => {
+  const canvasRef = useRef(null);
+  const niivue = useRef(null);
+
+  useEffect(() => {
+    niivue.current = new Niivue({ logLevel: "debug", ...nvOpts });
+=======
```

Comment:
@Sanjai-Shaarugesh , You need to remove the merge conflict strings that are still in the file (<<<HEAD and ==== lines)